PluralKit.Bot/Commands/SystemList.cs

```diff
@@ -18,6 +18,23 @@ public async Task MemberList(Context ctx, PKSystem target)
         // - RenderMemberList checks the indivual privacy for each member (NameFor, etc)
         // the own system is always allowed to look up their list
         var opts = ctx.ParseListOptions(ctx.DirectLookupContextFor(target.Id));
+        if (ctx.MatchFlag("raw")) {
+            var members = (await ctx.Database.Execute(conn => conn.QueryMemberList(target.Id, opts.ToQueryOptions())))
+                .ToList();
+            
+            var showId = ctx.MatchFlag("by-id") || ctx.MatchFlag("bid") || ctx.MatchFlag("id");
```

Comment:
generally, `by-x` flags are *sorting* options, not *display* options; either way this needs to use the options from `ParseListOptions`

also worth noting: you can pass multiple arguments to `Match` or `MatchFlag` instead of calling the method multiple times

Reply:
I made the flag checks use a single call to MatchFlag, and i also added a privacy check that *should* work. Also, I changed the option to "show-id" (shortened to "id") and "show-display-name" (shortened to "show-dn" or "dn")